docs/api/dialog.md

```diff
@@ -51,6 +51,8 @@ The `dialog` module has the following methods:
     untouched on Windows. For example, a button label of `Vie&w` will be
     converted to `Vie_w` on Linux and `View` on macOS and can be selected
     via `Alt-W` on Windows and Linux.
+  * `resolvesAliases` Boolean (optional) - Whether to resolve symlinks to their target path
```

Comment:
This feels like an option similar to `showHiddenFiles` so I'm wondering if this would be a better fit as a `properties` option.

What do you think?